fault/verilator_target.py

```diff
@@ -102,6 +103,12 @@ def run(self):
             prefix = str(verilog_file)[:-2]
             magma.compile(prefix, self.circuit, output="verilog")
         assert verilog_file.is_file()
+        if self.include_verilog:
```

Comment:
does verilator allow passing multiple file names? then we can just pass all the filenames without modifying this one. As long as we properly specify the top, it seems like this would work.

Reply:
I believe multiple files passed to verilator does indeed work as expected.